package_creator.py

```diff
@@ -54,10 +59,11 @@ def merge_script_package_to_yml(package_path, dir_name, dest_path=""):
 
     yml_text, script_path = insert_script_to_yml(package_path, script_type, yml_text, dir_name, yml_data)
     yml_text, image_path = insert_image_to_yml(dir_name, package_path, yml_data, yml_text)
+    yml_text, desc_path = insert_description_to_yml(dir_name, package_path, yml_data, yml_text)
 
     with io.open(output_path, mode='w', encoding='utf-8') as f:
         f.write(yml_text)
-    return output_path, yml_path, script_path, image_path
+    return output_path, yml_path, script_path, image_path, desc_path
```

Comment:
Did you check you are not breaking anything this way?

Reply:
What do you mean?

Reply:
check that no one is using this function outside the module (you changed the function interface)